prisma/schema.prisma

```diff
@@ -40,28 +34,184 @@ model Account {
-    id           String   @id @default(cuid())
+    id           String   @id @default(uuid())
     sessionToken String   @unique
     userId       String
     expires      DateTime
     user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
 }
 
 model User {
-    id            String    @id @default(cuid())
+    id            String       @id @default(uuid())
     name          String?
-    email         String?   @unique
+    email         String?      @unique
     emailVerified DateTime?
+    phone         String?
     image         String?
     accounts      Account[]
     sessions      Session[]
+    inspections   Inspection[]
+    createdAt     DateTime     @default(now())
+    updatedAt     DateTime     @updatedAt
+    roles         Role         @relation(fields: [roleId], references: [id], onDelete: Cascade)
+    roleId        String
 }
 
-model VerificationToken {
-    identifier String
-    token      String   @unique
-    expires    DateTime
+model Role {
+    id    String   @id @default(uuid())
+    name  String   @unique
+    users User[]
+    type  RoleType
 
-    @@unique([identifier, token])
+    createUser Boolean @default(false)
+    updateUser Boolean @default(false)
+    deleteUser Boolean @default(false)
+    readUser   Boolean @default(false)
+
+    createSite Boolean @default(false)
+    updateSite Boolean @default(false)
+    deleteSite Boolean @default(false)
+    readSite   Boolean @default(false)
+
+    createInspection Boolean @default(false)
+    updateInspection Boolean @default(false)
+    deleteInspection Boolean @default(false)
+    readInspection   Boolean @default(false)
+
+    createMachine Boolean @default(false)
+    updateMachine Boolean @default(false)
+    deleteMachine Boolean @default(false)
+    readMachine   Boolean @default(false)
+
+    createChecklist Boolean @default(false)
+    updateChecklist Boolean @default(false)
+    deleteChecklist Boolean @default(false)
+    readChecklist   Boolean @default(false)
+
+    createChecklistItem Boolean @default(false)
+    updateChecklistItem Boolean @default(false)
+    deleteChecklistItem Boolean @default(false)
+    readChecklistItem   Boolean @default(false)
+
+    createReport Boolean @default(false)
+    updateReport Boolean @default(false)
+    deleteReport Boolean @default(false)
+    readReport   Boolean @default(false)
 }
+
+enum RoleType {
+    INSPECTOR
+    MANAGER
+    GOD
+}
+
+model Site {
+    id         String       @id @default(uuid())
+    name       String       @unique
+    street     String
+    city       String
+    state      String
+    postalCode String
+    country    String
+    phone      String?
+    email      String?
+    userId     String
+    equipment  Machine[]
+    Inspection Inspection[]
+    createdAt  DateTime     @default(now())
+    updatedAt  DateTime     @updatedAt
+}
+
+model Machine {
+    id          String              @id @default(uuid())
+    name        String
```

Comment:
We will probably have taxonomies and or tags for machines when the business spec arrives.